# Generated by Django 2.2.4 on 2024-07-10 18:18

import api.models
from django.conf import settings
import django.core.validators
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('api', '0010_auto_20240709_1512'),
    ]

    operations = [
        migrations.AlterField(
            model_name='direccion',
            name='nombre',
            field=models.CharField(max_length=255, validators=[api.models.validate_only_letters]),
        ),
        migrations.AlterField(
            model_name='etiqueta',
            name='nombre',
            field=models.CharField(max_length=255, validators=[api.models.validate_only_letters]),
        ),
        migrations.AlterField(
            model_name='personal',
            name='apellido',
            field=models.CharField(max_length=100, validators=[api.models.validate_only_letters]),
        ),
        migrations.AlterField(
            model_name='personal',
            name='cedula',
            field=models.IntegerField(primary_key=True, serialize=False, unique=True, validators=[django.core.validators.MinValueValidator(1000000, message='La cédula debe tener al menos 8 dígitos.'), django.core.validators.MaxValueValidator(999999999, message='La cédula debe tener como máximo 9 dígitos.')]),
        ),
        migrations.AlterField(
            model_name='personal',
            name='nombre',
            field=models.CharField(max_length=100, validators=[api.models.validate_only_letters]),
        ),
        migrations.AlterField(
            model_name='ticket',
            name='resuelto_por',
            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
        ),
    ]
